### ISSUE TITLE
Fix submit reviews button (should be always active)

### ISSUE THREAD

**Issue by kacperknapik** [SAP/cloud-commerce-spartacus-storefront#4944]

**Comment by Zeyber** [comment]
QA: Submit button should not be disabled

**Comment by KateChuen** [comment]
Tested on dev10 (with the epic/accessibility branch), and theme calydon enabled. 
I still see the Submit button disabled. 

<img width="963" alt="Screen Shot 2019-11-18 at 11 04 10 AM" src="https://user-images.githubusercontent.com/45409515/69068721-422a6900-09f3-11ea-9c67-a7c6480c38f2.png">

**Comment by kacperknapik** [comment]
Just checked on dev17. It is actually working.

QA info: use dev17 for testing this.

**Comment by KateChuen** [comment]
I still see the button as disabled. I tried on dev3 and dev12 on chrome, safari & firefox.
I aso asked GC to check and he sees the button as disabled too. 

Caine and Kacper see the button as enabled though.
I asked @developpeurweb to try on his side... Waiting on his results now...

**Comment by developpeurweb** [comment]
This look fine on my end for all 3 browsers (screenshot below). I'm testing on my **local** and **dev3**.
I can confirm that it doesn't work in **dev12**, but it seems `calydon` is not enabled there (maybe it's that bug Brian pointed out about the order of the imports in the 'styles.scss' file).

![Screen Shot 2020-02-18 at 1 46 03 PM](https://user-images.githubusercontent.com/15113219/74771904-05257b00-525d-11ea-8a81-b9b91bb03d73.png)
![Screen Shot 2020-02-18 at 1 46 22 PM](https://user-images.githubusercontent.com/15113219/74771905-05be1180-525d-11ea-8c10-e84a4740a567.png)
![Screen Shot 2020-02-18 at 1 47 43 PM](https://user-images.githubusercontent.com/15113219/74771907-05be1180-525d-11ea-981c-274dad9840ca.png)

**Comment by KateChuen** [comment]
The fix must have been introduced in ticket #4962. After updating dev3 to get the latest changes, I can now see the "Submit" button for reviews as always active. 
Verified in chrome, firefox and safari.
